how-to/workspace-platform-starter/client/src/framework/launch.ts

```diff
@@ -385,33 +385,80 @@ async function launchView(viewApp: PlatformApp): Promise<PlatformAppIdentifier |
 			const platform = getCurrentSync();
 
 			if (!isEmpty(viewApp.launchPreference?.bounds) || !isEmpty(viewApp?.launchPreference?.options?.view)) {
-				const preferenceWindow = await platform.createWindow({
-					workspacePlatform: !isEmpty(viewApp.launchPreference?.options?.view?.hostUrl)
-						? {
-								windowType: "platform"
-						  }
+				let workspacePlatform:
+					| {
+							disableMultiplePages?: boolean;
+							windowType?: WindowType;
+							favicon?: string | undefined;
+							title?: string;
+							pages?: [
+								{
+									title?: string;
+									iconUrl?: string;
+									layout?: unknown;
+								}
+							];
+							toolbarOptions?: { buttons: [] } | undefined;
+					  }
+					| undefined = {};
+
+				const layout = {
+					content: [
+						{
+							type: "stack",
+							content: [
+								{
+									type: "component",
+									componentName: "view",
+									componentState: manifest
+								}
+							]
+						}
+					],
+					settings: {
+						hasHeaders: viewApp.launchPreference?.options?.view?.host?.hasHeader
```

Comment:
Rename to `hasHeaders` for consistency, as these are the Host preferences

Reply:
done